getpaid/views.py

```diff
@@ -75,5 +75,6 @@ def get_redirect_url(self, **kwargs):
             url_name = getattr(settings, 'GETPAID_FAILURE_URL_NAME', None)
 
         if url_name is not None:
-            return reverse(url_name, kwargs={'pk': self.payment.order_id})
+            # Unaweza specific thing - identify order by uuid that is not its pk
+            return reverse(url_name, kwargs={'uuid': self.payment.order.uuid})
```

Comment:
to o czym gadalismy: przełączanie pk/uuid za pomocą ustawienia globalnego django